lib/awspec/helper/finder/autoscaling.rb

```diff
@@ -14,6 +14,14 @@ def find_launch_configuration(id)
                                                                 })
         res.launch_configurations.single_resource(id)
       end
+
+      def select_block_device_mapping(id, device_id)
```

Comment:
@inokappa 
Could you rename `select_block_device_mapping` to `find_block_device_mapping` ?

Because this method return 1 resource.